store/store.go

```diff
@@ -192,8 +192,10 @@ func getStructFields(s interface{}) []string {
 	return fields
 }
 
+// Deltas enabled by default on classic, but allow opting in or out on both classic and core.
 func useDeltas() bool {
-	return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL")
+	deltasDefault := release.OnClassic
+	return osutil.GetenvBool("SNAPD_USE_DELTAS_EXPERIMENTAL", deltasDefault)
```

Comment:
snap-confine has a manual page that lists all the environment variables it respects. I think it would be good to craft a manual page for snapd as well. Let me know if I can help you get started.